app/uk/gov/hmrc/hecstubs/models/CompanyDetails.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hecstubs.models
+
+import play.api.libs.json.JsonConfiguration.Aux
+import play.api.libs.json.JsonNaming.SnakeCase
+import play.api.libs.json.{Json, JsonConfiguration, OWrites}
+
+final case class CompanyDetails(companyName: String)
+
+object CompanyDetails {
+
+  implicit val config: Aux[Json.MacroOptions]      = JsonConfiguration(SnakeCase)
```

Comment:
Earlier  i was thinking including the case class but it's not needed now in stubs. It will in hec-applicant-frontent. Will remove it from here. Just FYI, this is needed for mapping snake case to  camelcase in case class

Reply:
removed